### ISSUE TITLE
backup_databases.sh can't find database

### ISSUE THREAD

**Issue by at055612** [gchq/stroom-resources#89]
This line 
``` sh
source "$DIR"/config/stroom_core_test.env
```
needs to be before 
``` sh
readonly DATABASES=( \
  "${STROOM_AUTH_DB_NAME:-auth}"
  "${STROOM_DB_NAME:-stroom}"
  "${STROOM_STATS_DB_NAME:-stats}"
)
```

**Comment by at055612** [comment]
Needs to be fixed in 6.0 and merged up the chain

**Comment by at055612** [comment]
Fixed on 6.1 and merged up
